api/routes/socket.js

```diff
@@ -24,6 +24,7 @@ module.exports = function (app, server) {
 				break;
 			case 'socket/ticket/new':
 				socketController.addTicket(action.ticket);
```

Comment:
Hmm, not quite. If `addTicket` does error under some odd circumstances, the returned Promise is currently ignored, which will result in Node.js printing some weird errors. We have to explicitly ignore the error by attaching a `.catch(() => {})` no-op error handler (or use an `async`-`await` `try`-`catch` block with an empty `catch` clause).

Reply:
I feel kind of weird not doing anything in the catch block. In other areas don't we console.err stuff?

Reply:
I think using console.error helps to debug. Let me just add that in.

Reply:
@TimothyGu is this ok?